performance_profiling/profiling_train_base_nvtx.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>Could you please help remove these log?</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>